times_test.go

```diff
@@ -37,6 +37,17 @@ func TestPast(t *testing.T) {
 	}.validate(t)
 }
 
+func TestReltimeOffbyone(t *testing.T) {
+	testList{
+		{"1w-1", RelTime(time.Unix(0, 0), time.Unix(7*24*60*60, -1), "ago", ""), "6 days ago"},
+		{"1w", RelTime(time.Unix(0, 0), time.Unix(7*24*60*60, 0), "ago", ""), "1 week ago"},
```

Comment:
Heh, I've been using the go font for go code, so I don't remember what alignment looks like.  :)  Added, though.

Reply:
Apologies for missing that comment.  github's code review is a little foreign to me.  I'm used to the ones where you have to actually take action when someone calls something out (that is, if I ignore it, I have to at least do it intentionally).  It does seem easy to drop stuff here.

Not sure about the tests, though.  I'd've hoped a lot would break changing a `>=` to a `>`